dev.yml

```diff
@@ -7,7 +7,7 @@ up:
     - openssl
     - sqlite
   - ruby:
-      version: 2.2.3p172-shopify
+      version: 2.3.1
```

Comment:
Should not it be 2.3.3? Or greater? We use that in production, so it makes sense to use that here too.